src/components/MainNavBar/MainNavBar.tsx

```diff
@@ -16,6 +16,7 @@ import UserProfile from "./UserProfile";
 import UserAvatar from "./UserAvatar";
 import UMLSDialog from "./ULMSDialog";
```

Comment:
Do you mind renaming this component to `UMLSDialog`? If it causes too many issues, we can write it up as a tech debt story.

Reply:
@jkotanchik-SB Sure just did. Thanks so much Joe, it's a name hard to remember